src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -824,6 +824,41 @@ bool Dataloader::_load_map_dir(DefinitionManager& definition_manager) const {
 	return ret;
 }
 
+bool Dataloader::_load_song_chances(DefinitionManager& definition_manager) {
+	static constexpr std::string_view song_chance_file = "music/songs.txt";
+	const fs::path path = lookup_file(song_chance_file, false);
+
+	bool ret = true;
+	SongChanceManager& song_chance_manager = definition_manager.get_song_chance_manager();
+
+	if(song_chance_file.empty()) {
```

Comment:
```suggestion
	if (path.empty()) {
```
This needs to check the lookup up path, not the constant path which will never be empty.